web/packages/teleterm/src/services/pty/fixtures/mocks.ts

```diff
@@ -29,13 +29,21 @@ export class MockPtyProcess implements IPtyProcess {
 
   dispose() {}
 
-  onData() {}
+  onData() {
+    return () => {};
+  }
```

Comment:
I made it so that all `on(Event)` functions return cleanup functions, even though we call the cleanup functions only for a few of them.

---

I know that we discussed returning cleanup functions under the `cleanup` key of an object, but I'm still not sure about this approach. In my opinion it makes the function definition a little bit more noisy. Besides, what else would an `onEvent` function return? Usually they return `void`, so having a convention where they instead return a cleanup function instead makes sense to me.